haproxy/datadog_checks/haproxy/data/conf.yaml.example

```diff
@@ -34,7 +34,12 @@ instances:
 
     ## @param collect_aggregates_only - boolean - optional - default: true
```

Comment:
Should we change the type of `collect_aggregates_only` in the signature to string, since 'both' is also accepted ?

Feels a bit strange though. It's a boolean, but also accept 'both'.

Reply:
I'm in favor of keeping the boolean as this field accepts any "boolean-ish" value like (`y`, `yes`, `on`, `1`, `True`, `true`, `false`, `False` ....).

I agree it feels a bit strange but I think this is what makes the most sense.

Reply:
Sounds fair :)